pyvista/core/dataset.py

```diff
@@ -586,6 +586,183 @@ def translate(self, xyz: Union[list, tuple, np.ndarray]):
         """
         self.points += np.asarray(xyz)
 
+    def scale(self, xyz: Union[list, tuple, np.ndarray]):
+        """Scale the mesh.
+
+        Parameters
+        ----------
+        xyz : scale factor list or tuple or np.ndarray
+            Length 3 list, tuple or array.
+
+        Examples
+        --------
+        >>> import pyvista
+        >>> from pyvista import examples
+        >>> pl = pyvista.Plotter(shape=(1, 2))
+        >>> pl.subplot(0, 0)
+        >>> pl.show_axes()
+        >>> _ = pl.show_grid()
+        >>> mesh1 = examples.download_teapot()
+        >>> _ = pl.add_mesh(mesh1)
+        >>> pl.subplot(0, 1)
+        >>> pl.show_axes()
+        >>> _ = pl.show_grid()
+        >>> mesh2 = mesh1.copy()
+        >>> mesh2.scale([10.0, 10.0, 10.0])
+        >>> _ = pl.add_mesh(mesh2)
+        >>> pl.show(cpos="xy")
+        """
+        self.points *= np.asarray(xyz)
+
+    def flip_x(self, point=None, transform_all_input_vectors=False):
+        """Flip mesh about the x-axis.
+
+        Parameters
+        ----------
+        point : list, optional
+            Point to rotate about.  Defaults to origin ``(0.0, 0.0, 0.0)``.
+
+        transform_all_input_vectors : bool, optional
+            When ``True``, all input vectors are
+            transformed. Otherwise, only the points, normals and
+            active vectors are transformed.
+
+        Examples
+        --------
+        >>> import pyvista
+        >>> from pyvista import examples
+        >>> pl = pyvista.Plotter(shape=(1, 2))
+        >>> pl.subplot(0, 0)
+        >>> pl.show_axes()
+        >>> mesh1 = examples.download_teapot()
+        >>> _ = pl.add_mesh(mesh1)
+        >>> pl.subplot(0, 1)
+        >>> pl.show_axes()
+        >>> mesh2 = mesh1.copy()
+        >>> mesh2.flip_x()
+        >>> _ = pl.add_mesh(mesh2)
+        >>> pl.show(cpos="xy")
+        """
+        if point is None:
+            point = (0.0, 0.0, 0.0)
```

Comment:
I suggest we move these to be the default of the `point` parameter in all the new `flip` methods.

Reply:
Some filters default to the mesh center, and that would actually make sense here. Too bad the rotate_* filters don't do that already.

Reply:
```suggestion
        if point is None:
            point = self.center
```